public/locales/en.json

```diff
@@ -225,5 +225,7 @@
   "zap": "Zap in",
   "zapInAmount": "Zap in amount",
   "fees": "Fees",
-  "feesInfo": "Some explanation about where the fees come from"
+  "feesInfo": "Some explanation about where the fees come from",
+  "ZapInEdu": "What is Zap in?",
+  "ZapInEduDesc": "Zap in is a simplication of the normally complicated yield farming process. You can simply swap any token to a auto-compounding yield farming token in one click."
```

Comment:
```suggestion
  "ZapInEduDesc": "\"Zap in\" allows you to swap any token (like cUSD or CELO) for a share in a yield farm. Earn automatic compound interest with [10x lower fees](https://docs.revo.market/dyor/faqs#hasnt-someone-already-done-this) and no hassle. Learn more about DeFi on Revo [here](https://docs.revo.market/)"
```

Reply:
I felt like "simplication of the normally complicated yield farming process" says more about the status quo than about us. If people know about the status quo, they don't need this, and if they don't, it won't mean as much to them.